apps/nextjs/src/components/board/items/item-menu.tsx

```diff
@@ -115,6 +131,9 @@ export const BoardItemMenu = ({
         <Menu.Item leftSection={<IconCopy size={16} />} onClick={() => duplicateItem({ itemId: item.id })}>
           {tItem("action.duplicate")}
         </Menu.Item>
+        <Menu.Item leftSection={<IconMessageCircle2 size={16} />} onClick={openWidgetFeedbackModal}>
+          {tItem("action.feedback")}
+        </Menu.Item>
```

Comment:
Render this conditionally, only when analytics are enabled

Reply:
Technically this is not analytics. We are just leveraging our umami endpoint to send feedback. It won't send anything else other than that form

Reply:
Hmm... in theory you're right, but we disable umami when you disabled analytics.
For the sake of transparency and integrity, I think we should keep doing it that way, hence disabling the feedback menu item.

Reply:
@Meierschlumpf @SeDemal what's your opinion on this ?